x/datapool/client/cli/txContract.go

```diff
@@ -0,0 +1,94 @@
+package cli
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	wasmUtils "github.com/CosmWasm/wasmd/x/wasm/client/utils"
```

Comment:
```suggestion
	wasmutils "github.com/CosmWasm/wasmd/x/wasm/client/utils"
```
Let's name packages with only lowercase chars. https://go.dev/doc/effective_go#package-names

Reply:
modified in [abecf64](https://github.com/medibloc/panacea-core/pull/288/commits/abecf64bd84313eb2a8de05373eb08f76fe860c8)